### ISSUE TITLE
Unable to fully remove spicetify v2.1.0

### ISSUE THREAD

**Issue by MrMeepNL** [khanhas/spicetify-cli#606]
I'm trying to revert to version v1.2.1 but i am unable to do so because there is still something of spicetify v2.1.0 left on the system. I have been using the Powershell methods on Windows 10.

Removing the .spicetify & spicetify-cli folders, as well as re-installing spotify v1-1-39-612 (an older version) multiple times has had no effect;
`spicetify -v`
still returns `2.1.0`

I have tried this script from [Pastebin](https://pastebin.com/ntwNEfSC) (from this [video](https://www.youtube.com/watch?v=RdbN1ig3BNY));
`Add-Type -AssemblyName PresentationFramework
[System.Windows.MessageBox]::Show('Please download Spotify version 1.1.39.612 from https://spotify.en.uptodown.com/windows/download/2368262 and block updates. DO NOT LOG IN')`
`Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/khanhas/spicetify-cli/master/install.ps1" | Invoke-Expression`
`[System.Windows.MessageBox]::Show('Please copy the contents of the spicetify v1.2.1 zip file to spicetify-cli')`
`spicetify`
`spicetify backup apply enable-devtool`
`[System.Windows.MessageBox]::Show('Please drag the themes into the themes folder located at C:\Users\YourUserName\.spicetify\Themes') `
`cd "$(spicetify -c | Split-Path)\Themes\Dribbblish"`
`Copy-Item dribbblish.js ..\..\Extensions`
`spicetify config extensions dribbblish.js`
`spicetify config current_theme Dribbblish color_scheme base`
`spicetify config inject_css 1 replace_colors 1 overwrite_assets 1`
`spicetify apply`

I have also tried to force it to install v1.2.1 directly with this (with the Execution Policy = RemoteSigned);
`Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/khanhas/spicetify-cli/master/install.ps1" -OutFile $env:TEMP\install.ps1
Invoke-Expression "$env:TEMP\install.ps1 1.2.1"`

What else is there to delete? Path variables? something in \temp?

**Comment by khanhas** [comment]
When you run the `Invoke-Expression ` command, it should show which link it used to download release package. Can you check one more time if it was v1.2.1.

**Comment by MrMeepNL** [comment]
I entered
`Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/khanhas/spicetify-cli/master/install.ps1" -OutFile` `$env:TEMP\install.ps1 Invoke-Expression "$env:TEMP\install.ps1 1.2.1"`
and
`$env:TEMP\install.ps1 Invoke-Expression "$env:TEMP\install.ps1 1.2.1"`
both returned the same thing;
`DOWNLOADING    https://github.com/khanhas/spicetify-cli/releases/download/v1.2.1/spicetify-1.2.1-windows-x64.zip > OK`
`EXTRACTING     C:\Users\--\spicetify-cli\spicetify-1.2.1-windows-x64.zip into C:\Users\--\spicetify-cli > OK`
`REMOVING       C:\Users\--\spicetify-cli\spicetify-1.2.1-windows-x64.zip > OK`
 `> OK`
`Run spicetify --help to get started`.
after which `spicetify -v` returns
`2.1.0`
I did this again while removing the now present .spicetify and spiceify-cli folder. This did the same as above, it just recreated the spicetify-cli folder.

In both cases it seems to be downloading the v1.2.1 through the correct link.

**Comment by NoTextToSpeech** [comment]
@MrMeepNL

Delete the .spicetify and spicetify-cli folders and try installing spicetify like normal using
`Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/khanhas/spicetify-cli/master/install.ps1" | Invoke-Expression
`

Then once you install it, download  [V1.2.1](https://github.com/khanhas/spicetify-cli/releases/tag/v1.2.1) (pick whichever tar/zip you need for your system) and extract it to the spicetify-cli folder. Overwriting the files is fine.

After that, run the `spicetify` command to generate the .spicetify folder + refresh the spicetify version.

Confirm whether or not the version updated by running `spicetify -v`

I tried poking around with the install.ps1 but it always managed to install the newest version despite my poor use of coding knowledge to force it to attempt to install v1.2.1.

**Comment by MrMeepNL** [comment]
@NoTextToSpeech 
I tried what you said but it still returns `2.1.0`. I have made a discovery using 
`spicetify path assets` with the theme set to SpicetifyDefault

There is another spicetify inside the hidden C:/ProgramData folder,;
`C:\ProgramData\chocolatey\lib\spicetify-cli`
It seems my chocolatey install of spicetify I did a while back may have had effect.

I tried your method again with the chocolatey folder removed but to no avail;
`Cannot find file at '..\\lib\spicetify-cli\tools\spicetify.exe' (C:\ProgramData\chocolatey\lib\spicetify-cli\tools\spicetify.exe). This usually indicates a missing or moved file.`
Everything is still now sort of gone, even `choco uninstall spicetify-cli -n` returns errors stating it has already been uninstalled, but it is still referred to when reinstalled with your method.

I reinstalled spicetify-cli v1.1.0 through chocolatey, after which it did uninstall without errors.

Your method seems to work now. I still had some problems installing dribbblish-dynamic.js correctly, as it wasn't detecting the .js file when applying. Using your video I was able to fix this, thanks @NoTextToSpeech !

Also thanks @khanhas for helping me on my way and for making spicetify in the first place!

**Conclusion (for anyone seeing this thread at a later date)**

If you installed spicetify using `choco install spicetify-cli`, you first need to properly uninstall the choco version using `choco uninstall spicetify-cli` before using the method shown in [this video](https://www.youtube.com/watch?v=RdbN1ig3BNY&t=314s).
